SPECS/freefont/freefont.spec

```diff
@@ -11,6 +11,7 @@ Distribution:   Mariner
 Source0:        https://ftp.gnu.org/pub/gnu/%{name}/%{name}-ttf-%{version}.zip
 
 # BUG in Mariner RPMbuilder not providing zip
```

Comment:
Can we remove this comment while we're here? There's nothing wrong with not providing `unzip` in our worker chroot, it's a decision we made to keep our build environment lean.